docs/userguide/clustermanager/working-with-anp.md

```diff
@@ -25,31 +25,31 @@ git clone -b v0.0.24/dev https://github.com/mrlihanbo/apiserver-network-proxy.gi
 cd apiserver-network-proxy/
 ```
 
-### Step 2: Compile images
+### Step 2: Build images
 
-Compile the proxy-server and proxy-agent images.
+Build the proxy-server and proxy-agent images.
 
 ```shell
 docker build . --build-arg ARCH=amd64 -f artifacts/images/agent-build.Dockerfile -t swr.ap-southeast-1.myhuaweicloud.com/karmada/proxy-agent:0.0.24
 
 docker build . --build-arg ARCH=amd64 -f artifacts/images/server-build.Dockerfile -t swr.ap-southeast-1.myhuaweicloud.com/karmada/proxy-server:0.0.24
 ```
 
-### Step 3: Generate a certificate
+### Step 3: Generate certificates
 
-Run the command to check the IP address of karmada-host-control-plane:
+Run the command to check the IP address of Karmada control plane:
 
 ```shell
-docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' karmada-host-control-plane
+docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' karmada-host
```

Comment:
The name in the Kind cluster is used here:
```shell
# docker ps
CONTAINER ID   IMAGE                  COMMAND                  CREATED       STATUS       PORTS                       NAMES
64019301ec5e   kindest/node:v1.23.4   "/usr/local/bin/entr…"   3 weeks ago   Up 3 weeks   127.0.0.1:46403->6443/tcp   member3-control-plane
d92c3e0f70bf   kindest/node:v1.23.4   "/usr/local/bin/entr…"   3 weeks ago   Up 3 weeks   127.0.0.1:36033->6443/tcp   karmada-host-control-plane
67cadb8c6e38   kindest/node:v1.23.4   "/usr/local/bin/entr…"   3 weeks ago   Up 3 weeks   127.0.0.1:38029->6443/tcp   member2-control-plane
a409ef2865f7   kindest/node:v1.23.4   "/usr/local/bin/entr…"   3 weeks ago   Up 3 weeks   127.0.0.1:38501->6443/tcp   member1-control-plane
```

Reply:
Great, I see.